source/draw/gpu/opengl/shaders/yuvSemiplanarToBgra.frag

```diff
@@ -0,0 +1,19 @@
+#version 300 es
+precision highp float;
+uniform sampler2D texture0;
+uniform sampler2D texture1;
+in vec2 fragmentTextureCoordinate;
+out vec4 outColor;
+vec4 YuvToRgba(vec4 t)
+{
```

Comment:
I know this was only copied from another file, but should this bracket go on the previous line?